Makefile

```diff
@@ -40,11 +39,6 @@ clean:
 	rm -rf coverage
 	rm -rf build
 	rm -rf cover
-	make -C docs clean
 	python setup.py clean
 
-doc:
```

Comment:
What is the reason to remove `docs/`? I didn't find any mentions of it in the PR's description.

Reply:
It was a documentation from ``pyramid_sockjs`` and it never used before.